fastlane/Fastfile

```diff
@@ -228,6 +228,18 @@ end
 
 desc "Verifies all tests pass and the current state of the repo is valid"
 private_lane :validate_repo do
+  # Ensure that no ENV vars which interefere with testing are set
```

Comment:
Is the `Fastfile` the right spot to put this code? Shouldn't this be part of the test suite instead?